content/3. Procedurele-informatie/3. Processen/4. Implementatieproces/2. Bouwen softwaresysteem/Datatype-Declaratie.md

```diff
@@ -0,0 +1,52 @@
+---
+title: "[Hier een titel, begin met 1. Onderwerp]"
+taxonomie:
+  - "[taxco]"
+  - ib-19.2.Datatype.Declaratie.PI
+  - ib-19.3.Datatype.Declaratie.PI
+draft: true
+---
+
+> Meer informatie over [[1. Declaratie|1. Declaratie]]
+
+## Stappen in de Systematische Probleem Aanpak
+### 1. Doel
+Het doel van een declaratie is om aan te geven wat voor soort data een variabele bevat en hoe deze in het programma gebruikt wordt.
+
+### 2. Subdoelen
+Splits het probleem in subdoelen:
+
+- Welke soort informatie moet de variabele opslaan? (bijvoorbeeld een getal, tekst, of een complex object).
+- Waar moet de informatie van deze variabele beschikbaar zijn? (bijvoorbeeld binnen een functie, een klasse, of in het hele programma).
```

Comment:
() na de ?

Reply:
Opzich logisch toch? Mogelijke oplossing voor de vraag, of liever gewoon weghalen?

Reply:
Wel houden, maar taalkunding imo beter om het een geneste - te maken dus iets als:
- Welke soort informatie moet de variabele opslaan? 
   - Bijvoorbeeld: een getal, tekst, of een complex object

Reply:
hard